tasks/coverage/src/babel/mod.rs

```diff
@@ -167,12 +167,13 @@ impl Case for BabelCase {
     fn skip_test_case(&self) -> bool {
         let not_supported_plugins =
             ["async-do-expression", "flow", "placeholders", "decorators-legacy", "recordAndTuple"];
-        let has_not_supported_plugins = self
-            .options
-            .plugins
-            .iter()
-            .filter_map(Value::as_str)
-            .any(|p| not_supported_plugins.contains(&p));
+        let has_not_supported_plugins = self.options.plugins.iter().any(|p| match p {
+            Value::String(plugin_name) => not_supported_plugins.contains(&plugin_name.as_str()),
+            Value::Array(a) => a.iter().any(|plugin_name| {
+                plugin_name.as_str().is_some_and(|n| not_supported_plugins.contains(&n))
+            }),
+            _ => false,
+        });
```

Comment:
Good spot!

I think it's only the first element of the nested array which is the plugin name. e.g. [this one](https://github.com/babel/babel/blob/3bcfee232506a4cebe410f02042fb0f0adeeb0b1/packages/babel-plugin-transform-arrow-functions/test/fixtures/arrow-functions/self-referential/options.json):

```json
{
  "plugins": [
    ["transform-arrow-functions", { "spec": false }],
    "transform-function-name"
  ]
}
```

So this would do the trick:

```rs
Array(a) => a.first().is_some_and(|plugin_name| {
    plugin_name.as_str().is_some_and(|n| not_supported_plugins.contains(&n))
}),
```

Or, if we want to find out if there are any more configs which we're not understanding correctly:

```rs
let has_not_supported_plugins = self.options.plugins.iter().any(|p| {
    let plugin_name = match p {
        Value::String(plugin_name) => Some(plugin_name.as_str()),
        Value::Array(a) => a.first().and_then(|plugin_name| plugin_name.as_str()),
        _ => None,
    };
    let plugin_name = plugin_name.expect("Failed to parse plugins config");
    not_supported_plugins.contains(&plugin_name)
});
```

Reply:
Great, I prefer to second way